pkg/skaffold/kubernetes/debugging/container_manager_test.go

```diff
@@ -84,3 +84,11 @@ func TestContainerManager(t *testing.T) {
 		t.CheckDeepEqual(1, terminatedCount)
 	})
 }
+
+func TestContainerManagerZeroValue(t *testing.T) {
```

Comment:
this is so cool that you can have nil dereference checks from within the instance methods